cg-spec/editors_draft.html

```diff
@@ -415,12 +414,124 @@ <h2>Concepts and Abstract Syntax</h2>
+        triple can contain quoted triples. A triple can be used as an <em>asserted
+        triple</em> and as a <em>quoted triple</em> in the same graph.
+      </p>
+
+      <p>
+        A quoted triple is written in Turtle-star and related syntaxes using delimiters `<<` and `>>`.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            << :a :name "Alice" >>
+        -->
+      </pre>  
+      <p>
+        The definition of quoted triple is recursive. A quoted triple can itself have a
+        subject or object component which is another quoted triple. Cycles are not allowed
+        and can not be written in any RDF syntax. In this next example, there is a quoted
+        triple with property `:reportedBy` and whose subject is another quoted triple `:a
+        :name "Alice"`.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            << << :a :name "Alice" >> :reportedBy :charlie >>
+        -->
+      </pre> 
+    </section>
+
+    <section id="rdf-star-graph-examples">
+      <h2>RDF-star Graph Examples</h2>
+        
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            PREFIX : <http://www.example.org/> 
+
+            :bob :name "Bob" .
+            << :a :name "Alice" >> :statedBy :bob .
+              -->
+      </pre> 
+      <p>
+        In the example above, there is a quoted triple `:a :name "Alice"` which is used as subject for an asserted
+        triple. The example graph has two asserted triples:
+      </p>
+      <ol>
+        <li>`:bob :name "Bob" .`</li>
+        <li>`&lt;&lt; :a :name "Alice" &gt;&gt; :statedBy :bob .`</li>
+      </ol>
+      <p>
+        The graph does not have an asserted triple `:a :name "Alice"` because
+        the triple `:a :name "Alice"` is not an element of the set of triples.
+      </p>
+      <p>
+        A triple can be used as both an asserted triple and quoted triple in the same graph.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            PREFIX : <http://www.example.org/> 
+
+            :bob :name "Bob" .
+            :a :name "Alice" .
+            << :a :name "Alice" >> :statedBy :bob .
+              -->
+      </pre> 
+    </section>
+
+    <section id="annotation-syntax">
+      <h2>Annotation Syntax</h2>
+      <p>
+        <em>Annotation syntax</em> provides a convenience short cut.  It is used to both
+        assert a triple and also use the triple as a quoted triple as the subject of
+        further triples.  It uses delimiters `{|` and `|}` folliwing an asserted triple to
+        make that triple, as a quoted string, the subject of the enclosed RDF triples.
+      </p>
```

Comment:
Probably need to clarify that this syntax is available in Turtle, TriG, and SPARQL. Other serialization formats may have similar syntactic sugar, but it is not specifically required of RDF-star compatible syntaxes to support annotation.

Reply:
> <em>Annotation syntax</em> provides a convenience shortcut in Turtle-star and
>         related syntaxes.

SPARQL-star is a later - keep this text focused on RDF-star.

Reply:
I think it would be fine -- and desirable -- to say something very like "This syntax is available in Turtle-star and TriG-star serializations. (Also available in SPARQL-star, discussed in section xyz below)"